### DISCUSSION TITLE
Migrating from SQLite to PostgreSQL

### DISCUSSION THREAD

**Discussion by qstorey** [firefly-iii/firefly-iii#8878] [Firefly III]
Hi,

I have been running firefly-iii for a year now and I am super happy with it. I started out with a simple setup and opted for SQLite but now I'm looking to move to PostgreSQL for better performance.

I tried looking through the docs and existing discussions but couldn't find any hits. Are there any guides or tools for migrating firefly-iii from SQLite to PostgreSQL?

**Comment by JC5** [comment]
There are no instructions for Firefly III specifically, but there may be generic guides out there?

**Reply by qstorey** [reply]
I can have a look at generic migration tools for SQLite to PostgreSQL but I wanted to make sure first that the ORM or database layer in firefly-iii would cope with me migrating and changing database technology.

I don't want to lose any data or go into an unsupported mode of operating firefly-iii.

**Reply by JC5** [reply]
Ah yes. Both are supported, so a migration should not give you any trouble.
